jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/DisplayMessageActivity.java

```diff
@@ -10,7 +10,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_display_message);
+        setContentView(R.layout.display_message);
```

Comment:
The point was that the layout is now both used directly in an activity but also in a fragment, so that it might be a bit misleading calling it `activity_display_message` :) I'm not sure the rest is consistent though, I'll take a look :p